src/cache/precomputed_quantities.jl

```diff
@@ -369,37 +395,29 @@ function eddy_diffusivity_coefficient(C_E, norm_v_a, z_a, p)
 end
 
 """
-    set_precomputed_quantities!(Y, p, t)
+    set_implicit_precomputed_quantities!(Y, p, t)
```

Comment:
Can we also include which functions this impacts? For example: `implicit_tendencies!`, ....

Reply:
The quantities updated in `set_implicit_precomputed_quantities!` are required to compute both `implicit_tendency!` and `remaining_tendency!`; they are just handled implicitly (updated once per Newton iteration, rather than once per Newton solve). These quantities include everything related to velocity and thermodynamics, so they affect every tendency in our model.